app/serializers/story_serializer.rb

```diff
@@ -1,4 +1,10 @@
 class StorySerializer < ActiveModel::Serializer
   attributes :id, :project_id, :team_id, :name, :slug, :description, :owner_id, :created_at, :updated_at, :deleted_at
   has_one :owner
+
+  def owner
```

Comment:
it was when the story itself has been archived, and it blows up because it is trying to access `owner` on nil